modules/websocket-client/src/websocket.browser.ts

```diff
@@ -1,2 +1 @@
-// @flow
 export default WebSocket
```

Comment:
eslint catches an error because `browser` is not true in eslintrc.
Can you add a `eslintrc.js` in websocket-client with following contents?
```
module.exports = {
  env: { browser: true }
};
```